doc/mgmt/SONiC_Ztp_DesignDoc.md

```diff
@@ -0,0 +1,254 @@
+# Zero Touch Provisioning (ZTP)
+ZTP support through sonic-management framework
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [About This Manual](#about-this-manual)
+  * [Scope](#scope)
+  * [Definition/Abbreviation](#definitionabbreviation)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+
+# Revision
+| Rev  |    Date    |      Author       | Change Description |
+| :--: | :--------: | :---------------: | ------------------ |
+| 0.1  | 10/10/2019 | Arunsundar Kannan | Initial version    |
+
+# About this Manual
+This document provides general information about the ZTP support inside SONiC management framework.
+# Scope
+Covers Northbound interface for the ZTP feature, as well as Unit Test cases.
+
+# Definition/Abbreviation
+
+### Table 1: Abbreviations
+| **Term** | **Meaning**             |
+| -------- | ----------------------- |
+| ZTP      | Zero Touch Provisioning |
+
+# 1 Feature Overview
+
+Provide management framework capabilities to handle:
+
+- Show status of ztp service
+- Enable ztp service
```

Comment:
You could combine "Enable ZTP service" and "Disable ZTP service" into just "Toggle ZTP" service.

Also, please capitalize ZTP throughout the document where it is used as an abbreviation (i.e., everywhere except in code, CLI examples, etc.).